src/handlers.rs

```diff
@@ -405,7 +406,8 @@ where
     .and_then(|_| {
       cb();
       Ok(())
-    }).select(cancel_rx)
+    })
+    .select(cancel_rx)
```

Comment:
I think there’s some difference in formatting between versions of rustfmt 
Ideally we could fix a version...

Reply:
Could be, but if you look at the context of this change it looks correct.

Reply:
That said, I upgraded rustfmt from 0.8.2 to 0.99.1 and it does produce a different output, so I'm backing out the rust file changes.